cfme/utils/appliance/__init__.py

```diff
@@ -102,7 +102,15 @@ class ApplianceException(Exception):
     pass
 
 
-class IPAppliance(object):
+class BaseAppliance(object):
+    """
+        Represents base appliance class intended to hold base properties existing in every appliance
+
+    """
+    pass
```

Comment:
Shouldn't `type=None` be on the base appliance? Why is it on `IPAppliance`?

Reply:
agreed. fixed